pattern_import_export/models/ir_exports.py

```diff
@@ -284,7 +296,7 @@ def _generate_import_with_pattern_job(self, patterned_import):
             datas = self._read_import_data(attachment_data)
         except Exception as e:
             patterned_import.status = "fail"
-            patterned_import.info = _("Failed (check details)")
+            patterned_import.info = "Failed (check details)"
             patterned_import.info_detail = e
         res = (
```

Comment:
@sebastienbeau as we discussed before: the issue is that we presume that load([]) will always fail as we want it to fail, which is not the case. For example:
- We try to load example.users.fail.csv : our first column is id. Instead of putting an integer, we put a string. This will cause an error with psycopg2 (bad request in pg). This is not caught and will raise outside of the patterned.import.export.
- We try to load example.users.ok.csv with bad Pattern config: we change separators to something like %, then the entire first line will be considered one value (because we don't find any % in the line and then we have \n). So our first value is: "id,name,company_ids". This will raise too.

Reply:
I just push a fix and also created a PR for Odoo V14 : https://github.com/odoo/odoo/pull/60260